src/common/util/src/file_util.cpp

```diff
@@ -229,6 +229,21 @@ void ov::util::convert_path_win_style(std::string& path) {
     std::replace(path.begin(), path.end(), '/', '\\');
 }
 
+void ov::util::node_name_to_file_name(std::string& nodeName) {
```

Comment:
Let's add some description since we add new function to the core.

Reply:
added to header, for uniformity